tests/component/sections/dataset/dataset-publish/PublishDatasetModal.spec.tsx

```diff
@@ -79,10 +81,12 @@ describe('PublishDatasetModal', () => {
           repository={repository}
           persistentId="testPersistentId"
           releasedVersionExists={true}
+          nextMajorVersion={'2.0'}
+          nextMinorVersion={'1.1'}
           handleClose={handleClose}
         />
       )
-      cy.findByText('Update Current Version').should('exist')
+      cy.contains('Update Current Version').should('exist')
     })
   })
   describe('PublishDatasetModal', () => {
```

Comment:
In this file we have three `describe('PublishDatasetModal'`, we should only have one same describe with different cases inside.